### ISSUE TITLE
export map to gcs

### ISSUE THREAD

**Issue by angel-r** [gee-community/ee-runner#24]
Are there any examples for exporting map to gcs using ee-runner?

I tried this, without success

`
// Load the global Accessibility to Cities image.
var accessibility = ee.Image('Oxford/MAP/accessibility_to_cities_2015_v1_0');

// Color palette for visualizing accessibility data.
var accessibilityPalette = ['f2fef8', 'defce1', 'c9f3bc', 'cbeca7', 'd6e793',
  'e2d87b', 'd4a561', 'c46c49', 'ab3a38', '922f4b', '7d285d', '672069',
  '3a1453', '1b0c3c', '050526', '00030f', '000000'];

// Apply the color palette to the log of travel time.
var accessibilityVis = {min: 0, max: 10, palette: accessibilityPalette};
var logAccessibility =
    accessibility.where(accessibility.gt(0), accessibility.log());
var accessibilityRgb = logAccessibility.visualize(accessibilityVis);

// Composite onto a solid-color background to fill in the oceans.
var background = ee.Image(0).visualize({palette: ['11101e']});
var accessibilityBlended = background.blend(accessibilityRgb).updateMask(1);

// Check the visualization.
Map.addLayer(accessibilityBlended, {}, 'accessibilityBlended');

// Define an export region.
var exportRegion = ee.Geometry.Rectangle([34, -3, 40, 1]);
Map.centerObject(exportRegion);
Map.addLayer(exportRegion, {}, 'exportRegion');

var params = {
  region: exportRegion,
  image: accessibilityBlended.clip(exportRegion),
  type:'EXPORT_TILES',
  description:'test22',  
  //crs:'EPSG:4326',
  maxZoom: 13,
  outputBucket: 'caminos'
}

var taskId = ee.data.newTaskId(1);

ee.data.startProcessing(taskId, params);`

**Comment by gena** [comment]
What do you mean by 'without success', any log? 

There is one example to do this, but a bit low-level https://github.com/gee-community/ee-runner/blob/master/examples/export_image_gcs.js, as an alternative, you can try https://github.com/google/earthengine-api/blob/master/javascript/src/batch.js#L179

**Comment by angel-r** [comment]
I did not log anything, but i can try with batch,but I tried with batch and it worked

// Load the global Accessibility to Cities image.
var accessibility = ee.Image('Oxford/MAP/accessibility_to_cities_2015_v1_0');

// Color palette for visualizing accessibility data.
var accessibilityPalette = ['f2fef8', 'defce1', 'c9f3bc', 'cbeca7', 'd6e793',
  'e2d87b', 'd4a561', 'c46c49', 'ab3a38', '922f4b', '7d285d', '672069',
  '3a1453', '1b0c3c', '050526', '00030f', '000000'];

// Apply the color palette to the log of travel time.
var accessibilityVis = {min: 0, max: 10, palette: accessibilityPalette};
var logAccessibility =
    accessibility.where(accessibility.gt(0), accessibility.log());
var accessibilityRgb = logAccessibility.visualize(accessibilityVis);

// Composite onto a solid-color background to fill in the oceans.
var background = ee.Image(0).visualize({palette: ['11101e']});
var accessibilityBlended = background.blend(accessibilityRgb).updateMask(1);

// Check the visualization.
Map.addLayer(accessibilityBlended, {}, 'accessibilityBlended');

// Define an export region.
var exportRegion = ee.Geometry.Rectangle([34, -3, 40, 1]);
Map.centerObject(exportRegion);
Map.addLayer(exportRegion, {}, 'exportRegion');


 var mytask = ee.batch.Export.map.toCloudStorage({
	writePublicTiles: false,
    image: accessibilityBlended.clip(exportRegion),
    description: 'export-example3',
	path: 'test3',
    bucket: 'caminos',  
    maxZoom: 13,
	//region: exportRegion
    region: JSON.stringify(exportRegion.toGeoJSON())
});

var taskId = ee.data.newTaskId(1);

ee.data.startProcessing(taskId, mytask.config_);

var list = ee.data.getTaskList();
console.log(list);
